src/systems/filecoin_blockchain/chainsync/_index.md

```diff
@@ -345,22 +347,23 @@ State Machine:
 - Validation computation is considerable, and a serious DOS attack vector.
 - Secure implementations must carefully schedule validation and minimize the work done by pruning blocks without validating them fully.
 - `ChainSync` SHOULD keep a cache of unvalidated blocks (ideally sorted by likelihood of belonging to the chain), and delete unvalidated blocks when they are passed by `FinalityTipset`, or when `ChainSync` is under significant resource load.
-- It is key to note that any block received after the `ROUND_CUTOFF` time must be automatically discarded by the miner until the start of the next epoch.
+- It is key to note that any block received after the `EPOCH_CUTOFF` time must be automatically discarded by the miner until the start of the next epoch.
```

Comment:
@ZenGround0 touch this up per notes we shared, related to potential DoSing issue. Specifically it is a weaker rule: no block received after the cutoff should be considered for the HEAD. But it should not be discarded. Same as with 294 (no blocks received `Epoch_cutoff` time after start are considered...)